flight/flight.py

```diff
@@ -1,52 +1,60 @@
 """Launch code for flight state machine"""
 import asyncio
+import logging
+
 from mavsdk import System
 import mavsdk as sdk
-from .states import STATES, State
+
+from flight.states import STATES, State
+
+SIM_ADDR: str = "udp://:14540"  # Address to connect to the simulator
+CONTROLLER_ADDR: str = "serial:///dev/ttyUSB0"  # Address to connect to a pixhawk board
+
+
+class DroneNotFoundError(Exception):
+    """Exception for when the drone doesn't connect"""
+
+    pass
 
 
 class StateMachine:
-    """ 
+    """
     Initializes the state machine and runs the states.
-      
-    Attributes: 
+
+    Attributes:
         current_state (State): State that is currently being executed.
         drone (System): Our drone object; used for all flight functions.
     """
 
     def __init__(self, initial_state: State, drone: System) -> None:
-        """ 
-        The constructor for StateMachine class. 
-  
-        Parameters: 
+        """
+        The constructor for StateMachine class.
+
+        Parameters:
             current_state (State): State that is currently being executed.
             drone (System): Our drone object; used for all flight functions.
         """
         self.current_state: State = initial_state
         self.drone: System = drone
 
     async def run(self) -> None:
-        """ 
+        """
         Runs the state machine by infinitely replacing current_state until a
         state return None.
         """
         while self.current_state:
             self.current_state: State = await self.current_state.run(self.drone)
 
 
```

Comment:
If the while loop ends, would it be good to ensure the drone is landed before just stopping the state machine?

Reply:
It's not really the responsibility of the state machine and also out of scope for this PR